src/win/tty.c

```diff
@@ -138,6 +140,10 @@ int uv_tty_init(uv_loop_t* loop, uv_tty_t* tty, uv_file fd, int readable) {
     /* shared between all uv_tty_t handles. */
     EnterCriticalSection(&uv_tty_output_lock);
 
+    /* Remember the original console text attributes. */
+    if (uv_tty_default_text_attributes == 0)
+      uv_tty_default_text_attributes = screen_buffer_info.wAttributes;
```

Comment:
screen_buffer_info.wAttributes **can** be zero, but as a a practical matter, it never **will** be.  Zero would mean black text on a black background, and nobody sets a console to that.  If for some reason a console was set this way, nothing bad would happen.  tty would just fall back to it's prior behavior, and assume Windows default colors.  I didn't see a reason to create a separate "uv_tty_default_text_attributes_is_valid" boolean.